### PR TITLE
[FEAT] Add End game socket event listener to Component

### PR DESCRIPTION
**Pull request by David-Xia0**
After the countdown, user will be moved to the swiping page (its currently not implemented)
After another 10 seconds (Dummy value) user will be forced to the results screen.

### CODE REVIEW

File: frontend/src/components/7.Swiping/SwipingPage.js [hunk 1]
```diff
@@ -31,8 +36,16 @@ function SwipingPage() {
...
   useEffect(() => {
     document.title = 'Yes or No?';
+    SocketEvents.endGame(socketContext.socket, goNextPge);
   }, []);
 
+  /**
+   */
+  const goNextPge = () => {
+    console.log('Game has Ended');
+    history.push('/Result');
```
Comment (Tianrens) [inline frontend/src/components/7.Swiping/SwipingPage.js:46]: replace here? since we don't want user to go back to the swiping stage?

**Review by Tianrens — DISMISSED** [review]

**Review by Tianrens — APPROVED** [review]
lgtm

**Comment by BlinkCreator** [comment]
Looks good

**Review by BlinkCreator — APPROVED** [review]
Good job David

**Pull request merged**